### PR TITLE
Minor pixel tracking

### PR DESCRIPTION
**Pull request by zyskowsk**
Going off of @pius's work in #32, add method to tracker to generate tracking urls for pixel tracking.

### CODE REVIEW

File: lib/mixpanel-ruby/consumer.rb [hunk 1]
```diff
@@ -123,7 +123,7 @@ def send(type, message)
     # [response code, response body]
     #
     # as the result of the response. Response code should be nil if
-    # the request never recieves a response for some reason.
+    # the request never receives a response for some reason.
```
Comment (joeatwork) [inline lib/mixpanel-ruby/consumer.rb]: Nice!

**Comment by joeatwork** [comment]
Sweet!

File: lib/mixpanel-ruby/tracker.rb [hunk 2]
```diff
@@ -131,5 +131,42 @@ def alias(alias_id, real_id, events_endpoint=nil)
 
       ret
     end
+
+    # A call to #generate_tracking_url will return a formatted url for
+    # pixle based tracking.  #generate_tracking_url takes a distinct_id
```
Comment (zakj) [inline lib/mixpanel-ruby/tracker.rb]: s/pixle/pixel/; extra space before `#generate_tracking_url`

File: lib/mixpanel-ruby/tracker.rb [hunk 3]
```diff
@@ -131,5 +131,42 @@ def alias(alias_id, real_id, events_endpoint=nil)
...
+    # representing the source of that event (for example, a user id),
+    # an event name describing the event, and a set of properties describing
+    # that event. Properties are provided as a Hash with string keys and
+    # strings, numbers or booleans as values. For more information, please see:
+    # https://mixpanel.com/docs/api-documentation/pixel-based-event-tracking
+    #
+    #     tracker = Mixpanel::Tracker.new
+    #
+    #     # generate pixle tracking url in order to track that user
```
Comment (zakj) [inline lib/mixpanel-ruby/tracker.rb]: pixel

**Comment by pius** [comment]
Nice work @zyskowsk and thanks for merging, @joeatwork!

**Pull request merged**